specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-30/disk.json

```diff
@@ -1862,6 +1866,28 @@
         ]
       }
     },
+    "DiskEncryptionSetType": {
+      "type": "string",
+      "description": "The type of key used to encrypt the data of the disk.",
+      "enum": [
+        "EncryptionAtRestWithCustomerKey",
+        "EncryptionAtRestWithPlatformAndCustomerKeys"
+      ],
+      "x-ms-enum": {
+        "name": "DiskEncryptionSetType",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "EncryptionAtRestWithCustomerKey",
+            "description": "Disk is encrypted at rest with Customer managed key that can be changed and revoked by a customer."
```

Comment:
descriptions need to be updated to something like Resources using the disk encryption set would be encrypt at rest ...

Reply:
Updated.